src/modules/esl-utils/misc/uid.ts

```diff
@@ -1,10 +1,11 @@
+const eslSequencesKey = Symbol('__esl_sequences__');
```

Comment:
For 4th version, we still use fallback
```suggestion
const eslSequencesKey = (window.Symbol || String)('__esl_sequences__');
```
Also we can simplify variable name and consider it as a constant e.g. `KEY` or `SEQUENCE_KEY`

Reply:
SEQUENCE_KEY - it's a good idea

Reply:
If we use ```(window.Symbol || String)('__esl_sequences__');``` we get  "TS7015: Element implicitly has an 'any' type because index expression is not of type 'number'" and "TS1169: A computed property name in an interface must refer to an expression whose type is a literal type or a 'unique symbol' type" errors. I am not sure how to handle them quickly and we currently use the same syntax ```/** Key to store listeners on the {@link SyntheticEventTarget} instance*/
const LISTENERS = Symbol('_listeners');``` in target.ts. If it is really important I`ll spend some more time handling errors.

Reply:
To resolve type issue declare it explicietly as unique symbol
```
const SEQUENCE_KEY: unique symbol = ((window.Symbol || String)('__esl_sequences__')) as any;
```